examples/gradle-kt-dsl/run-tests.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+
+# Retry the build for 30 minutes, every 10 seconds
+NUMBER_OF_RETRIES=180
```

Comment:
if the build time between retries is nonzero, wouldn't this be substantially more than 30m?